test/localstack.ts

```diff
@@ -0,0 +1,234 @@
+import * as cxapi from "@aws-cdk/cx-api";
```

Comment:
Why did this file get changed? Have you merged from main

Reply:
Woops, removed.